packages/server/dataloader/atlassianLoaders.ts

```diff
@@ -35,22 +35,24 @@ export const freshAtlassianAuth = (parent: RethinkDataLoader) => {
       const results = await Promise.allSettled(
         keys.map(async ({userId, teamId}) => {
           const atlassianAuth = await getAtlassianAuthByUserIdTeamId(userId, teamId)
-
-          if (!atlassianAuth?.refreshToken) {
-            // Not always an error! For suggested integrations, this won't exist.
-            // sendToSentry(new Error('No atlassian access token exists for team member'), {
-            //   userId,
-            //   tags: {teamId}
-            // })
+          if (!atlassianAuth) {
             return null
           }
+
           const {accessToken: existingAccessToken, refreshToken} = atlassianAuth
           const decodedToken = existingAccessToken && (decode(existingAccessToken) as any)
           const now = new Date()
           const inAMinute = Math.floor((now.getTime() + 60000) / 1000)
           if (!decodedToken || decodedToken.exp < inAMinute) {
-            const manager = await AtlassianServerManager.refresh(refreshToken)
-            const {accessToken, refreshToken: newRefreshToken} = manager
+            const {
+              accessToken,
+              refreshToken: newRefreshToken,
+              error
+            } = await AtlassianServerManager.refresh(refreshToken)
+            if (error) {
+              sendToSentry(new Error(error))
```

Comment:
This is questionable, do we want to know about the fact it failed? 🤔

Reply:
i don't think we've ever seen an error come from this, so sure!

Reply:
eh, sure! if we wanted to do it right we'd make the return type a union of a success | error since the return object can't have both an error & an access_token, but 🤷 

https://www.oauth.com/oauth2-servers/access-tokens/access-token-response/